CONTRIBUTING.md

```diff
@@ -1,62 +1,33 @@
-     representations.
-1. Create a GitHub PR. A reviewer should be with you in 24 hours.
-   - If a reviewer does not review within that time, please ping @rofrankel or
-     @toumorokoshi.
-   - Leave comments on any parts of the text that you would like to discuss.
-1. Come say hi in our various communication channels documented in the
-   [README](README.md#learn-and-connect).
-
-Some tips:
-
-- Less is more. If there's a hard disagreement about particular guidance,
-  consider filing a follow-up issue to get the agreed upon guidance merged.
-- Perfect is the enemy of good. Feel free to submit PRs that are mostly there,
-  and leave questions so reviewers can help address any questions or concerns.
+We welcome patches and contributions to this project.
 
-## Development Environment
+## Learn and Connect
 
-If you are contributing AEP content (rather than code) and want to be able to
-view it in your browser, the easiest way to do so is to run the site-generator
-script.
+If you'd like to get involved in the AEP community, we'd love to have you! The
+following channels of communication are available:
+
+- [The #aep channel in the CNCF Slack](https://cloud-native.slack.com/archives/C04TX46UCTV).
+  Sign up at https://slack.cncf.io.
+- We hold a weekly project meeting on Fridays. See [the AEP Google Calendar, to view any upcoming meetings](https://calendar.google.com/calendar/u/0/embed?src=7e35dd3df194a32f4fa7c03a3454e24ba3651e6f568248efe1ddf1e34a7bee9e@group.calendar.google.com&ctz=America/Los_Angeles) 
+for details. Options for other times, such as more EU-friendly meetings, can also be arranged. 
 
-We use [GitHub Pages][1] to make this documentation available, and a specific
-[site generator][2] to build the site.
+## Editing AEP Rules
 
-Run `./scripts/serve.sh` to:
+If you spot a minor issue that seems to be a mistake, such as a typo, please make an PR with a fix.
 
-- clone the site-generator repository.
-- run the appropriate npm commands.
-- begin the dev server at port 4321.
+For any substantive change or new rule proposal, it is best to search the [discussions](https://github.com/aep-dev/aeps/discussions) 
+for similar topics before opening a new one. Lighter weight discussions also happen in #aep channel 
+in the CNCF Slack is another place or in our weekly Friday calls for quick validation of the concept.
 
-### Hot reloading
+Once there is a clear point of view articulated and some amount of support gathered, the next step 
+would be to open a PR to reference the discussion and describe the work to be done. Any subsequent 
+pull request should then reference the issue and will enter the queue for review.
 
-The development server currently does not support hot reloading.
+## Development Environment
+
+If you are contributing AEP content (rather than code) and want to be able to
+view it in your browser, the easiest way to do so is to use our [site-generator project](https://github.com/aep-dev/site-generator).
```

Comment:
actually aep.dev also have a serve.sh to clone the appropriate repos for you: https://github.com/aep-dev/aeps/blob/main/scripts/serve.sh